lib/http.module.ts

```diff
@@ -13,14 +13,10 @@ import {
   HttpModuleOptionsFactory,
 } from './interfaces';
 import axiosRetry from 'axios-retry';
-import axiosBetterStacktrace from 'axios-better-stacktrace';
 
 const createAxiosInstance = (config?: HttpModuleOptions) => {
   const axiosInstance = Axios.create(config);
   axiosRetry(axiosInstance, config);
-  if(config?.isBetterStackTraceEnabled === undefined || config?.isBetterStackTraceEnabled) {
```

Comment:
Is this no longer needed? stack trace doesn't look nice in my solution, maybe because of this...